### ISSUE TITLE
Accessibility - Individual Goods Screen - Incorrect Content Reading Order

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#44]
TalkBack reads ‘Exploitation Type’, then ‘Produced with exploitative labor in “XX” countries’, then the drop down list of exploitation types. Not sure why talkback keeps reading items out of order. 

XX = countries that have exploited children for that good.

**Comment by aquilent-dgemoets** [comment]
We should be able to change the traversal order via code.

**Comment by jfc3-dol** [comment]
Thanks, for correcting this and I will close it.
